guides/v3.20.0/components/helper-functions.md

```diff
@@ -318,3 +318,66 @@ Hello, {{@person.firstName}} {{@person.lastName}}
 ```
 
 To consult all available built-in helpers, you can check the [template helpers API documentation](https://api.emberjs.com/ember/3.20/classes/Ember.Templates.helpers/).
+
+### The `in-element` helper
+
+Using the [`{{in-element}}`](https://api.emberjs.com/ember/3.27/classes/Ember.Templates.helpers/methods/in-element?anchor=in-element) helper, you can render content into a specific DOM element that is in a different part of the page. For instance, we might want
```

Comment:
For `v3.20.0`-`v3.26.0`, can you link to the corresponding version?

```suggestion
Using the [`{{in-element}}`](https://api.emberjs.com/ember/3.20/classes/Ember.Templates.helpers/methods/in-element?anchor=in-element) helper, you can render content into a specific DOM element that is in a different part of the page. For instance, we might want
```

Reply:
Yup!